pkgs/applications/networking/cluster/k3s/patches/0002-Add-nixpkgs-patches.patch

```diff
@@ -1,21 +1,19 @@
-From 1891be119e298ef7c86347a6b74d40ff539cf06e Mon Sep 17 00:00:00 2001
-From: Euan Kemp <euank@euank.com>
-Date: Sun, 31 May 2020 19:02:37 -0700
-Subject: [PATCH 2/2] Add nixpkgs patches
-
-These patches let us re-use the upstream build scripts when building for
-nix.
----
- scripts/build       | 20 ++++----------------
- scripts/package-cli | 10 ++++++----
- 2 files changed, 10 insertions(+), 20 deletions(-)
-
+-Subject: [PATCH 2/2] Add nixpkgs patches
+-Original patch by: Euan Kemp <euank@euank.com>
+-Adapted by: superherointj
```

Comment:
Probably doesn't matter but I would keep the original attributes and just add a `Co-authored-by: superherointj`.

Reply:
The original information does not match how current patch was generated (commit revision and dates won't match). Information on lines changed is unavailable. I have manually added this header.

Reply:
I wouldn't worry much about the headers here; nixpkgs git history has the canonical authorship info, and it just happens that git format-patch with email headers is the easiest way to copy around and apply patch files.

What you've got looks good to me!